### PR TITLE
created entities and changed batch jobs to run with new entities

### PR DESCRIPTION
**Pull request by fmandelli**
Please, review code related to issue #12

### CODE REVIEW

File: findagig-batch/src/main/java/findagig/source/entity/Event.java [hunk 1]
```diff
@@ -93,4 +93,19 @@ public void setArtists(final List<Artist> artists) {
     public enum Type {
         CONCERT, FESTIVAL
     }
+
+    @Override
+    public String toString() {
+        return "Event{" +
```
Comment (diegoicosta) [inline findagig-batch/src/main/java/findagig/source/entity/Event.java]: Bad practice here ... (String concatenation)
A better alternative would be use `StringBuilder`.
When asking for Intellij to create toString, it offers some builtin ways to do it.

Comment (fmandelli) [reply findagig-batch/src/main/java/findagig/source/entity/Event.java]: ok. All Entities have been reviewed and modified. String concatenation has been replaced by StringBuilder.

File: findagig-batch/src/main/java/findagig/batch/infra/SongKickConsumer.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package findagig.batch.infra;
+
+import org.springframework.web.client.RestTemplate;
+
+public class SongKickConsumer {
```
[the remark this answers is not in the document]
Comment (fmandelli) [reply findagig-batch/src/main/java/findagig/batch/infra/SongKickConsumer.java]: SonkGigConsumer class has been deleted. New methods related to SongKick API are going to be created within SongKickDriver

File: findagig-batch/src/main/java/findagig/batch/job/GigsProcessor.java [hunk 3]
```diff
@@ -1,11 +1,64 @@
...
+    }
+
+
+    /**
+     * Parses a musical Event from SongKick to either a single or multiple Gigs,
+     * depending on the number of artists participating in a such Event.
+     * @param ev is an Event object from SongKick
+     * @return a List of Gig objects
+     */
+    private List<Gig> parse(Event ev) {
```
[the remark this answers is not in the document]
Comment (fmandelli) [reply findagig-batch/src/main/java/findagig/batch/job/GigsProcessor.java]: GigsFactory Interface and SongKickGigsFactory class have been created to solve this issue. Please, review them and provide feedback. Thanks

[the remark this answers is not in the document]
Comment (diegoicosta) [reply findagig-batch/src/main/java/findagig/batch/job/GigsProcessor.java]: Bad name, but ok ...  (The dependency with song kick must be only inside Its driver)

**Pull request merged**